### ISSUE TITLE
reassigned tickets

### ISSUE THREAD

**Issue by ValWood** [pombase/canto#2781]
When a  curation session is reassigned, who gets an e-mail?

**Comment by kimrutherford** [comment]
> When a curation session is reassigned, who gets an e-mail?

The new curator should get an email.  Is that not working?

**Comment by ValWood** [comment]
Probably, but historically I am sure that I used to get an e-mail saying x has been reassigned to y.
Maybe it was because I did the reassignment myself, rather than the author doing it? But I haven't seen one of these reassignment e-mails for ages.

**Comment by jseager7** [comment]
@ValWood I was looking into Canto's email code for an unrelated reason and it looks like the expected behaviour is that the user who reassigns the session is the one who gets an email notification confirming the fact they reassigned the session.

This is based on the documentation in the code:
https://github.com/pombase/canto/blob/d3b36d876742d3b5d58959a036913c2f40168c76/lib/Canto/EmailUtil.pm#L81

So if you weren't the one doing the reassignment, you shouldn't have received an email confirming the reassignment.

But if you did reassign the session and didn't get an email confirming it, then something is probably wrong.

**Comment by ValWood** [comment]
Right, when an assignee reassigns the session to another person I only see this in the daily e-mail. It will appear as a "reassigned session"
